app/controllers/admin/registrations_controller.rb

```diff
@@ -6,12 +6,17 @@ class RegistrationsController < Admin::BaseController
 
     def index
       authorize! :show, Registration.new(conference_id: @conference.id)
-      @pdf_filename = "#{@conference.title}.pdf"
-      @registrations = @conference.registrations.includes(:user).order('registrations.created_at ASC')
-      @attended = @conference.registrations.where('attended = ?', true).count
+      @registrations = @conference.registrations.eager_load(
+        :qanswers, user: :roles
+      ).order('registrations.created_at ASC').to_a
```

Comment:
why is it needed to convert it to array? :confused:

Reply:
ActiveRecord emulates many of the functions in Enumerable, but performs additional queries when doing so; e.g. ActiveRelation#count performs a "SELECT count(*) FROM" whereas Enumerable#count is performed on the loaded collection without another roundtrip to the database.

Reply:
`@registrations` is redefined her for perfomance; I think you're confusing it with `@attended` which is defined on the next line :arrow_down: as `@registrations.count(&:attended)`.